conf/messages

```diff
@@ -21,4 +36,47 @@ unauthorised_non_agent.link = sign up to use software to submit your VAT returns
 
 session_expired.title = For your security, this service has been reset
 session_expired.heading = For your security, this service has been reset
-session_expired.guidance = The details you have given have been deleted because you did not continue the service for 15 minutes. 
\ No newline at end of file
+session_expired.guidance = The details you have given have been deleted because you did not continue the service for 15 minutes.
+
+common.agent.changeClient.youCan = You can
+common.agent.changeClient.link = change another client’s details
+common.agent.yourClient = your client
+common.agent.confirmationLetter = We will send a confirmation letter to the agency address registered with HMRC within 15 working days.
+common.agent.updateClient1 = We will also contact
+common.agent.updateClient2 = with an update.
+common.agent.changeClient = Change client
+
+# Choose return frequency dates messages
+chooseDatesForm.frequency.missing = Select the new VAT Return dates
+
+return_frequency.title = Choose the new VAT Return dates
+return_frequency.heading = Choose the new VAT Return dates
+return_frequency.current = The VAT Return dates are currently
+return_frequency.January = January, April, July and October
+return_frequency.February = February, May, August and November
+return_frequency.March = March, June, September and December
+return_frequency.Monthly = Every month
+
+confirm_return_frequency.title = Confirm the new VAT Return dates
+confirm_return_frequency.heading = Confirm the new VAT Return dates
```

Comment:
This isn't being used as it's the same as the message above

Reply:
This is being used. Line 34 of the confirm_dates view